test/testplatform.cpp

```diff
@@ -181,6 +182,16 @@ class TestPlatform : public TestFixture {
         ASSERT_EQUALS(64, platform.long_long_bit);
     }
 
+    void valid_config_native() const {
```

Comment:
Could you also please add a test for the actual issue you encountered. That would make it much clearer what is intended here.

Reply:
We should first decide whether `_T()` etc. are tied to a platform or a library. Then we can test if it is properly replaced. Btw, there are more calls to `isWindows()` which are also affected with `native`.

Reply:
It is neither. It is tied to a user(!) defined macro.

It needs to be put into a library though as it comes from the API/headers and not the compiler.

We need to review all code but for now it should be fine to make the proposed change. That's why we have a development cycle so we are allowed to break things temporarily. And it is also why I did it immediately at the beginning of it so we have the time we need. For now I would just like to see a test for the actual problem you encountered.

Reply:
I just noticed that we even have platform-dependent types in windows.cfg (e.g. `TCHAR`).